_develop/collections.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Work with Collections (Preview)
+{:.no_toc}
+
+Collections is the feature designed to make data portable among Blockstack applications. Sharing is accomplished by storing a user's data in a standard, structured way in a known Gaia location. Collections associate the data with a user's distributed ID. When users move among apps, the same data is available to each application. 
+
+On this page, you learn what collections are and how to use them. You'll learn about the `Contacts` collection in particular. The following topics are covered:
+
+* TOC
+{:toc}
+
+{% include note.html content="This is a preview release of the <code>Contact</code> collections is a preview release. This release allows developers to try out the new collections functionality and we are interested in collecting feedback. Please feel free to report issues or request enhancements with collections or <code>Contacts</code> themselves on the <a href='https://github.com/blockstack/blockstack-collections/issues/new' target='_blank'>blockstack/blockstack-collections</a> repository.  If you encounter problems with <code>blockstack.js</code> you can <a href='https://github.com/blockstack/blockstack.js/issues/new' target='_blank'>file issues or request enhancements on its repo</a>." %}
+
+## Understand how collections work
+
+One of Blockstack's goals is to give users true data ownership by enabling *data portability*.  Data portability allows users to login with their digital ID on any app and have access to the same data. For example, if a user adds a photo of a Hawaiian vacation in one app, that photo enters the user's data pool. Then, when the user opens a second app, that same photo is already in use because the user's digital ID login is identical for both apps. 
```

Comment:
We might want to consistently use decentralized ID instead of digital ID. All online IDs are digital IDs.

Consider changing the last part to:
"... that same photo will be available for use because data is linked to the same decentralized ID."

Reply:
Consistency in place. I restructured the sentence for clarity.